pxtblocks/blocklycompiler.ts

```diff
@@ -256,6 +256,11 @@ namespace pxt.blocks {
         }
     }
 
+    function getLoopBlockField(b: Blockly.Block) {
```

Comment:
getLoopFieldBlock?

Reply:
Helper to get the variable field block depending on type of loop block. The new blocks have it in a shadow block VAR, while older blocks have it in a field on the block. 

Maybe call it getLoopVariableField ?